.github/workflows/mvn-verify-check.yml

```
@@ -30,7 +30,44 @@ env:
     -Dmaven.scaladoc.skip
 
 jobs:
+  cache-dependencies:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8']
+    steps:
+      - uses: actions/checkout@v3 # refs/pull/:prNumber/merge
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+      - run: |
```

give this step some name so it is easier to navigate the workflow in the UI

```suggestion
      - name: populate-daily-cache
        run: |
```